helpers/browser/driver.js

```diff
@@ -232,6 +238,59 @@ class ChromeProtocol {
       });
     });
   }
+
+  beginNetworkCollect() {
+    return this.getInstance().then(chrome =>
+      new Promise((resolve, reject) => {
```

Comment:
oh weird, so... because this is on a newline I was thinking this was a multiline => which requires a explicit return...

since this is on a newline, I vote add the `{ } and the return` to avoid ambiguity